src/cls/rgw/cls_rgw_ops.h

```diff
@@ -554,23 +554,16 @@ WRITE_CLASS_ENCODER(rgw_cls_obj_check_mtime)
 
 struct rgw_cls_usage_log_add_op {
   rgw_usage_log_info info;
-  rgw_user user;
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(2, 1, bl);
+    ENCODE_START(3, 1, bl);
     ::encode(info, bl);
-    ::encode(user.to_str(), bl);
     ENCODE_FINISH(bl);
   }
 
   void decode(bufferlist::iterator& bl) {
-    DECODE_START(2, bl);
+    DECODE_START(3, bl);
     ::decode(info, bl);
-    if (struct_v >= 2) {
-      string s;
-      ::decode(s, bl);
-      user.from_str(s);
-    }
```

Comment:
@mikulely doesn't look like you addressed comment by @cbodley. In the decode instead of the `if (struct_v >= 2)` block you need to have the following:
```
if (struct_v == 2) {
  string s;
  ::decode(s, bl);
}
```